### PR TITLE
feat: add street view source parameter

### LINKED ISSUE #1260: Add StreetViewSource parameter to fetchStreetViewData
**Is your feature request related to a problem? Please describe.**
I'm currently using the `fetchStreetViewData` function in a class called `StreetViewUtils` to check if there is street detail information for a specific location within a library. However, the function lacks the ability to specify a `StreetViewSource` parameter, which is essential for my use case.

**Describe the solution you'd like**
I would like to request the addition of a `StreetViewSource` parameter to the `StreetViewUtils.fetchStreetViewData` function. This addition would greatly enhance the functionality of the function and better serve my requirements. Simply put, I need the ability to specify the source for the street view data, and this parameter addition would address that need. Thank you.

Check street view optional parameters: https://developers.google.com/maps/documentation/streetview/request-streetview#optional-parameters

### PR DESCRIPTION
**Pull request by yusufonderd**
Added street view source parameter to `fetchStreetViewData`

Fixes #1260  🦕

### CODE REVIEW

**Comment by yusufonderd** [comment]
Changed pr title as semantic types.

File: library/src/main/java/com/google/maps/android/StreetViewUtil.kt [hunk 1]
```diff
@@ -35,14 +35,24 @@ class StreetViewUtils {
         /**
          * This function will check whether a location is available on StreetView or not.
          *
-         * @param latLng Location to check
-         * @param apiKey Maps API Key
-         * @return A Status value specifying if the location is available on Street View or not,
```
Comment (wangela) [inline library/src/main/java/com/google/maps/android/StreetViewUtil.kt:40]: Should revert the line that deletes the docs for the return value.

File: library/src/main/java/com/google/maps/android/StreetViewUtil.kt [hunk 2]
```diff
@@ -35,14 +35,24 @@ class StreetViewUtils {
...
-         * whether the used key is a right one, or any other error.
+         * @param latLng The `LatLng` object representing the location for which you want to fetch Street View data.
+         * @param apiKey The API key for Google Maps services.
+         * @param source The source of the Street View panorama. It can be one of the following values:
```
Comment (wangela) [inline library/src/main/java/com/google/maps/android/StreetViewUtil.kt:40]: Indicate that the source parameter is optional because it has a default value

**Review by wangela — CHANGES_REQUESTED** [review]

File: library/src/main/java/com/google/maps/android/StreetViewUtil.kt (hunk 2 above)
Comment (yusufonderd) [reply library/src/main/java/com/google/maps/android/StreetViewUtil.kt]: Done ✅

File: library/src/main/java/com/google/maps/android/StreetViewUtil.kt (hunk 1 above)
Comment (yusufonderd) [reply library/src/main/java/com/google/maps/android/StreetViewUtil.kt:40]: Done ✅

**Review by wangela — APPROVED** [review]

**Pull request merged**